preliz/distributions/beta.py

```diff
@@ -165,9 +165,15 @@ def mean(self):
         return self.alpha / (self.alpha + self.beta)
 
     def mode(self):
-        return np.where(
-            (self.alpha > 1) & (self.beta > 1), (self.alpha - 1) / (self.alpha + self.beta - 2), 0.5
-        )
+        if self.alpha == 1 and self.beta == 1:
```

Comment:
We support parameters to be vectors, this will fail in those cases. You should explicitly write a vectorized form or use numba to vectorize a version working with scalars

Reply:
Oops...Completely forgot about that.
Fixed it :)